src/Conference/ConferenceDetails.jsx

```diff
@@ -18,8 +19,15 @@ export const StyledDescription = styled.p`
   padding: 0 100px 50px 100px;
 `;
 
-const ConferenceDetails = ({ conference, id, isLoading, error }) => {
+export const ConferenceDetails = ({
+  conference,
+  id,
+  isLoading,
+  error,
+  allCookies = {}
+}) => {
   const history = useHistory();
+  const sessionToken = allCookies.sessionToken;
```

Comment:
Unlike my previous comment, this is okay as sessionToken will be `undefined` here opposed to an error